portfolio/src/main/webapp/js/script.js

```diff
@@ -10,4 +10,31 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+/*
+    Filters the gallery based on the tab clicked. filterGallery() adds or removes show-gallery-img class 
+    from each of the images in order to show or hide it.
+*/
+filterGallery('all');
+function filterGallery(galleryFilter) {
+    var documents = document.getElementsByClassName("gallery");
```

Comment:
Per Google standards, we prefer `let` to `var`. The documentation doesn't explain this super well, but here's a stackoverflow page with more details! The general gist that var creates global variables, which is more error prone.

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var#:~:text=let%20allows%20you%20to%20declare,function%20regardless%20of%20block%20scope.

Google's style guide, if you're curious: go/js-style#features-use-const-and-let